src/views/CavaticaBixView.js

```diff
@@ -112,19 +124,31 @@ const CavaticaBixView = ({
           </Header>
         )}
 
-        {showModal === 'addProject' && (
+        {history.location.hash === '#add-cavatica-project' && (
```

Comment:
I think we can abstract this functionality out to work for all modals. By using the Semantic Modal `open` prop with a hook that checks the url hash against the modal name to set the open property instead of just conditionally rendering modals on a per page basis. 

#### pseudo code 
> src/modals/index.js  OR src/modals/KFModal.js
```js
const hashOpenHook = (history, modalName) => {
   const [open, setOpen] = useState(false);
   const modalNameHash = modalName.replace(' ','-').toLowerCase()
   return modalNameHash === history.location.hash 
}

... on open the modal also needs to push to url hash 

<Modal open={hashOpenHook()} />
```

> view where modal is used 
```js
<Modal />
```

This will also help tie all modals into the history stack so that the browsers back button works as expected